src/dlgTriggerEditor.cpp

```diff
@@ -3697,77 +3710,86 @@ void dlgTriggerEditor::saveAlias()
 
             if( pT->state() )
             {
-                if( old_name == "New Alias" )
+                if( old_name.compare( tr( "New Alias" ) ) )
```

Comment:
:bug: Logic inversion error - `QString::compare()` returns __zero__ == `(bool)false` if the strings ARE the same...